pennylane/fermi/fermionic.py

```diff
@@ -147,15 +147,16 @@ def __add__(self, other):
         method"""
 
         self_fs = FermiSentence({self: 1.0})
+        use_jax = qml.math.get_interface(other) == "jax"
 
         if isinstance(other, FermiSentence):
             return self_fs + other
 
         if isinstance(other, FermiWord):
             return self_fs + FermiSentence({other: 1.0})
 
-        if isinstance(other, (Number, ndarray)):
-            if isinstance(other, ndarray) and qml.math.size(other) > 1:
+        if isinstance(other, (Number, ndarray)) or use_jax:
```

Comment:
Cannot we just use `isinstance` to determine if the input data is a jax array?

Reply:
No, you would have to import JAX to do that e.g.`import jax.array`

Better to use qml.math.get_interface in this case.